resources/migrations/20210517073525-category-for-catalogue-item.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE category (
+  id serial NOT NULL PRIMARY KEY,
+  title varchar(256) NOT NULL,
+  start timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  endt timestamp NULL DEFAULT NULL
+);
```

Comment:
This is a quick "first iteration idea" I have. Any further suggestions @Macroz @opqdonut ?

Reply:
I don't think we need the start & end. Some tables have them for legacy reasons but we don't have a use case for expiring categories, right?

Also, shouldn't the category titles be localized?

Reply:
I'm not sure which part to comment because there is so much work in progress still but for this part we are transitioning more towards a document store style so I'd prefer if this were in that style.

So instead of title have a column like "data" and then a localized title there. And any other attributes we think of.

I also don't think we need start or end times for categories.

Reply:
thanks @opqdonut and @Macroz !